src/config/swagger.config.js

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
```

Comment:
Please remove it from the folder put it in `src`, delete the `config` folder and name the file `swagger.js`